elements/lisk-dpos/src/dpos.ts

```diff
@@ -84,6 +90,8 @@ export class Dpos {
 		this.delegateActiveRoundLimit = 3;
 		this.chain = chain;
 		this.rounds = new Rounds({
+			genesisHeight,
+			initRound,
 			blocksPerRound: this._delegatesPerRound,
```

Comment:
I notice or used somewhere else as `roundLength` we should be consistent. If required can create an issue to make these consistent.